kubectl

```diff
@@ -19,18 +19,19 @@ if [ ! -f "$DEFAULT_CLIENT" ]; then
 	downloadClient "$DEFAULT_VERSION"
 fi
 
-if [ "$1" == "config" ]; then
-	$DEFAULT_CLIENT "$@"
-	exit $?
```

Comment:
I would keep config with default client to make sure we can always configure our kubectl and dont have to wait 30 seconds for an i/o timeout if cluster not reachable for example.